core/src/main/scala/org/genivi/sota/core/WebService.scala

```diff
@@ -98,8 +98,13 @@ class UpdateRequestsResource(db: Database, resolver: ExternalResolverClient, upd
   import CirceMarshallingSupport._
 
   implicit val _db = db
+  val extractUUID = refined[Uuid](Slash ~ Segment)
+  val extractVin : Directive1[Vehicle.IdentificationNumber] = refined[Vehicle.Vin](Slash ~ Segment ~ PathEnd)
   val route = pathPrefix("updates") {
-    (get & refined[Uuid](Slash ~ Segment ~ PathEnd)) { uuid =>
+    (get & extractVin) { vin =>
+      complete(db.run(UpdateSpecs.getPackagesQueuedForVin(vin)))
```

Comment:
Let's not use the same path for two things. How about `/vehicles/VIN/queued`? And update the routing in webserver...